doc/early-access.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Early Access"
+date: 2023-09-04
+description: "Beta-testing the new API"
+weight: 2
+---
+
+As per the [API Transition
+Timeline](https://kernelci.org/blog/posts/2023/api-timeline/), the
+Early Access phase is from 4th September to 4th December 2023.  It
+allows users to request an account on the new KernelCI API for
+beta-testing purposes.
+
+There are still lots of incomplete or missing features with the new
+API & Pipeline.  It now has a [production-like
+deployment](https://github.com/kernelci/kernelci-api/tree/main/kube/aks)
+and can already run a minimalist pipeline with KUnit, a kernel build
+and a boot test on QEMU.  The aim of the Early Access phase is to make
+all the adjustments to the design that are necessary before reaching
+full production status.  So please give it a go and create issues on
+GitHub and ask questions on the [mailing
+list](mailto:kernelci@lists.linux.dev), IRC `#kernelci` on libera.chat
+or [Slack](https://kernelci.slack.com) with whatever you may find.
+
+## Requesting a user account
+
+The number of users and available resources are limited, so while we'll try and
+provide an account for every request we can't give any guarantee at this stage.
+Anyone interested is still very much encouraged to request an account of
+course, the more users the better.
+
+It can be done in two simple steps:
+
+* Create an "API Early Access" [issue on
+  GitHub](https://github.com/kernelci/kernelci-project/issues/new/choose)
+* Wait for an email confirmation of your user account which should contain a
+  randomly-generated password
+
+> **Tip**: If you don't have a GitHub account, please send an email to
+    [kernelci-sysadmin@groups.io](mailto:kernelci-sysadmin@groups.io) instead.
+
+## Setting up the `kci` tool
+
+While you're waiting for your user account to be created, you can already set a
+few things up that don't require any account.  Several parts of the API are
+publicly available for anonymous users and some commands can be run without any
+API at all.
+
+* The `kci` command line tool is available via a Docker image
+  `kernelci/kernelci`.  It typically relies on a local settings file
+  `kernelci.toml` which is specific to each user.  Here's a simple way to run a
+  container with your own settings file:
+
+```sh
+$ mkdir kernelci
+$ cd kernelci
+$ echo -e "[DEFAULT]\napi_config = \"early-access\"" > kernelci.toml
+$ docker pull kernelci/kernelci
+$ docker run -v $PWD:/home/kernelci -it kernelci/kernelci /bin/bash
+kernelci@3215c7c7b590:~$ kci api hello
+{"message": "KernelCI API"}
+```
+
+> **Tip**: If you're wondering how it managed to find the API, take a look at
+> `/etc/kernelci/core/api-configs.yaml` from within the Docker container.
+
+* Once you've received your confirmation email with your randomly-generated
+  password, you should change it to use your own arbitrary one instead:
+
+```sh
+$ kernelci@d7e27fe0ba0b:~$ kci user change_password --user <your-user-name>
```

Comment:
minor: should the host prompt (`^$ `) be kept with the docker one?

I don't have strong preference for or against it - I like both differentiation from the first listing as well as the below ones being clean and concise

Reply:
Good point.  I think it's important to make it clear what is run directly on the host and what is in the container.  Let's see if I can clarify that, maybe like I did in the AKS README with first a step to start the container than assume the rest of the commands are all run from it, with some explanation.

Reply:
OK I've added a small paragraph about it, please let me know if this makes sense.

Reply:
(also removed a long container prompt as a result)